static/src/stylesheets/layout/new-header/_feedback-form.scss

```diff
@@ -19,6 +19,12 @@
 
     @include mq(wide) {
         width: $gs-column-width * 3;
+
+        .has-page-skin & {
+            width: auto;
+            position: relative;
+            left: 0;
```

Comment:
Only to even out the `right` setting https://github.com/guardian/frontend/pull/17329/files/dc758479884d87a22568e4a03b1a8ce12e086123#diff-35c6508f9586373237605727b5a950a3R17

😆